plugins/ports-plugin/README.md

```diff
@@ -0,0 +1,15 @@
+# Ports Plug-in
+This plug-in is checking network and detect when new ports are listening or closed.
+
+If a process is listening, it will check
+- if listening interface is 127.0.0.1/::1 it asks the user to create a redirect for this port so it will externally available
+- if listening remotely:
+     - if there is a workspace server that is matching for this port, display the URL of this server and propose to open the link.
+     - if there is no workspace server matching, propose the user to do a port redirect.
+
+
+Port redirect mechanism:
```

Comment:
That means that some ingress or routes are pre-created when starting the workspace? That's a smart way to solve the problem though :-)

Reply:
@l0rd yes correct. There is a need of a 'pool' or routes that can be later used when need to proxify a port.

Reply:
Really cool